File: src/main/scala/grp/faq/entity/Answer.scala
package grp.faq.entity

import java.util.Date

import scala.beans.BeanProperty


class Answer() {
  @BeanProperty
  var cAnsid = 0L: Long
  @BeanProperty
  var cUid = 0: Byte
  @BeanProperty
  var cQid = 0L: Long
  @BeanProperty
  var cAnsTime: Date = null
  @BeanProperty
  var cAnsContext: String = null
  @BeanProperty
  var cIsDelete = 0: Byte
  @BeanProperty
  var cLastUpdate: Date = null
  @BeanProperty
  var cIsFold = 0: Byte

  def this(cAnsid: Long, cUid: Byte, cQid: Long, cAnsTime: Date, cAnsContext: String, cIsDelete: Byte, cLastUpdate: Date, cIsFold: Byte) {
    this()
    this.cAnsid = cAnsid
    this.cUid = cUid
    this.cQid = cQid
    this.cAnsTime = cAnsTime
    this.cAnsContext = cAnsContext
    this.cIsDelete = cIsDelete
    this.cLastUpdate = cLastUpdate
    this.cIsFold = cIsFold
  }
}

File: src/main/scala/grp/faq/entity/Answer.java
package grp.faq.entity;

import java.util.Date;

public class Answer {
    public Long cAnsid;

    public Byte cUid;

    public Long cQid;

    public Date cAnsTime;

    public String cAnsContext;

    public Byte cIsDelete;

    public Date cLastUpdate;

    public Byte cIsFold;

    public Answer(Long cAnsid, Byte cUid, Long cQid, Date cAnsTime, String cAnsContext, Byte cIsDelete, Date cLastUpdate, Byte cIsFold) {
        this.cAnsid = cAnsid;
        this.cUid = cUid;
        this.cQid = cQid;
        this.cAnsTime = cAnsTime;
        this.cAnsContext = cAnsContext;
        this.cIsDelete = cIsDelete;
        this.cLastUpdate = cLastUpdate;
        this.cIsFold = cIsFold;
    }

    public Answer() {
        super();
    }

    public Long getcAnsid() {
        return cAnsid;
    }

    public void setcAnsid(Long cAnsid) {
        this.cAnsid = cAnsid;
    }

    public Byte getcUid() {
        return cUid;
    }

    public void setcUid(Byte cUid) {
        this.cUid = cUid;
    }

    public Long getcQid() {
        return cQid;
    }

    public void setcQid(Long cQid) {
        this.cQid = cQid;
    }

    public Date getcAnsTime() {
        return cAnsTime;
    }

    public void setcAnsTime(Date cAnsTime) {
        this.cAnsTime = cAnsTime;
    }

    public String getcAnsContext() {
        return cAnsContext;
    }

    public void setcAnsContext(String cAnsContext) {
        this.cAnsContext = cAnsContext == null ? null : cAnsContext.trim();
    }

    public Byte getcIsDelete() {
        return cIsDelete;
    }

    public void setcIsDelete(Byte cIsDelete) {
        this.cIsDelete = cIsDelete;
    }

    public Date getcLastUpdate() {
        return cLastUpdate;
    }

    public void setcLastUpdate(Date cLastUpdate) {
        this.cLastUpdate = cLastUpdate;
    }

    public Byte getcIsFold() {
        return cIsFold;
    }

    public void setcIsFold(Byte cIsFold) {
        this.cIsFold = cIsFold;
    }
}